### PR TITLE
update on Add three icons

### PR DESCRIPTION
**Pull request by MichellDranig**
Removed width and height on icons

issue: #271

### CODE REVIEW

File: src/assets/svg/4G.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg width="41" height="40" viewBox="0 0 41 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 41 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (razvandk) [inline src/assets/svg/4G.svg:1]: 🤪 41, is this intended?

Comment (MichellDranig) [reply src/assets/svg/4G.svg:1]: it was like that but good question 😅

Comment (MichellDranig) [reply src/assets/svg/4G.svg]: Should be changed now

**Review by razvandk — APPROVED** [review]

File: src/assets/svg/4G.svg [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-  <path d="M7.56407 24.762L10.6781 20.82L10.0301 24.762H7.56407ZM9.14807 30H12.7121L13.1081 27.606H14.2241L14.7101 24.762H13.5941L14.7821 17.58H10.3001L4.48607 24.834L4.01807 27.606H9.54407L9.14807 30Z" fill="currentColor"/>
-  <path d="M20.9712 12.9156C23.1027 12.0368 25.5618 12.1019 27.6673 13.3175C29.7721 14.5327 31.0588 16.6303 31.3635 18.9156" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-  <path d="M19.0001 8.33013C22.5526 6.86555 26.6511 6.9739 30.1604 9C33.6683 11.0253 35.8128 14.5213 36.3206 18.3301" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (grandorf) [inline src/assets/svg/4G.svg:1]: still has width and height attributes in this file

**Review by grandorf — CHANGES_REQUESTED** [review]

File: src/assets/svg/4G.svg (hunk 2 above)
Comment (razvandk) [reply src/assets/svg/4G.svg]: i updated it ;-)

**Review by grandorf — APPROVED** [review]

**Pull request merged**